2.Invariants/contracts/FixedInterestOnlyLoans.sol

```diff
@@ -49,6 +49,9 @@ contract FixedInterestOnlyLoans {
     }
 
     function loanExists(uint256 instrumentId) public view returns (bool) {
+        if (instrumentId >= loans.length) {
```

Comment:
Without this change, `loanExists` function would revert on some calls, which would result in some invariants passing partially [vacuously](https://docs.certora.com/en/latest/docs/confluence/pitfalls.html#vacuity). It's still an issue for other view functions but we'll work on getting rid of the problem.